doc/nrf/app_power_opt.rst

```diff
@@ -0,0 +1,303 @@
+      :scale: 50 %
+      :alt: Online Power Profiler simulation for 40-byte payload with 20 minutes interval
+
+Observe that the average current is 48.93 µA, which is below the calculated maximum limit of 57 µA.
+
+3. As the next step, export the settings from the Online Power Profiler tool and compare it with the measurements in real networks. Complete the following sub-steps to export the settings from Online Power Profiler:
+
+   a. Click :guilabel:`Export settings` to store current Online Power Profiler settings to a :file:`.json` file.
+   #. Click :guilabel:`Export NCS project config` to export the configuration parameters in a :file:`opp.conf` file that can be used when building the firmware.
+
+Real-time power measurement using Power Profiler Kit II
+-------------------------------------------------------
+
+After exporting the settings from Online Power Profiler, the next step is to use the settings while measuring current on an nRF9160 DK.
+To measure current on an nRF9160 DK using the Power Profiler Kit II, while is connected to a live network , complete the following steps:
+
+1. Connect Power Profiler Kit II to an nRF9160 DK as described in `Power Profiler Kit II user guide <Power Profiler Kit II (PPK2)_>`_.
+#. Navigate to the :ref:`udp` sample in |NCS| and copy the contents of the :file:`opp.conf` file that was exported from Online Power Profiler into the :file:`prj.conf` file in the UDP sample folder (samples\\nrf9160\\udp).
+#. :ref:`Build and program the sample onto the nRF9160 DK <gs_programming>`.
+#. If PSM is available in your network, observe that the power measurement using Power Profiler Kit II produces the following results:
+
+   .. figure:: images/app_power_opt_ppk_psm.png
+      :scale: 50 %
+      :alt: Power Profiler Kit II sample output with PSM
+
+
+   If PSM is activated, you can see that the current value drops to a few µA.
+   As long as the PSM mode is activated and no other activity occurs, the current value remains in the PSM floor as shown in the above image until a transmission occurs or until the PSM timer times out. Even though the requested time out value in the use case is 60 minutes, it is decided by the network.
+   The PSM floor current is 4.69 µA as seen in the above image.
+
+#. Since the data transfer interval is 20 minutes, you can increase the frequency of transmission. Set :option:`CONFIG_UDP_DATA_UPLOAD_FREQUENCY_SECONDS` to ``120`` in :file:`prj.conf`.
+#. Re-build and program the sample.
+#. Observe that a 40-byte data packet is sent every two minutes. Make sure that the frequency is sufficiently higher than the duration of RRC inactivity present in the network. RRC inactivity timer value can be over one minute in some networks, and to get correct measurements, you must ensure that the device has gone into PSM before the new data packet is sent. After a duration of two minutes, observe the following results in the Power Profiler Kit II interface:
```

Comment:
This is the first time `RRC inactivity` is mentioned, and it's probably a new term to many users. Might point to an explanation or include it somewhere in this document